package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "chart.js",
+  "version": "3.0.0-beta.3",
```

Comment:
something requires the version to be placed after name?

Reply:
Oops, that was my error. Initially I tried removing the version but that causes `npm ci` to fail. I added it back in the wrong spot